tests/test_template2.py

```diff
@@ -0,0 +1,238 @@
+from os import scandir
+from typing import Generator
+
+import pytest
+
+from notifications_utils.template2 import render_notify_markdown
+
+
+# Skip all tests in this module.
```

Comment:
This should be a TODO #213 here. I won't insist on it. The benefit of TODO is that most IDEs will look for those and report them out when queried.

Reply:
I added a TODO.